tests/qos/qos_sai_base.py

```diff
@@ -1291,8 +1291,10 @@ def populateArpEntries(
 
         dut_asic = duthost.asic_instance(enum_frontend_asic_index)
 
-        dut_asic.command('sonic-clear fdb all')
-        dut_asic.command('sonic-clear arp')
+        if 'dualtor' in tbinfo['topo']['name']:
```

Comment:
Can we make it conditional for T2 only since we are seeing issue on T2 chassis ?

Reply:
thanks for the comment. Incorporated it ,please check